src/database/helpers.rs

```diff
@@ -278,3 +283,14 @@ pub async fn insert_or_update_address<'e>(
 
     Ok(result_map)
 }
+
+#[instrument(skip(postgres))]
+pub async fn get_account_names_stats(
+    postgres: &PgPool,
+) -> std::result::Result<AccountNamesStats, sqlx::error::Error> {
+    let query = "SELECT COUNT(name) FROM names AS count";
```

Comment:
You don't need so specify the name?

```suggestion
    let query = "SELECT COUNT(*) FROM names AS count";
```

Reply:
I believe by specifying (limiting) the field Postgres will try to use the index, since this column is in the index, while `*` will force to seek.

Reply:
@geekbrother you can verify if the query indeed benefits from the index by running the query planner like so:

```sql
EXPLAIN ANALYZE SELECT COUNT(name) FROM names;
```
and see if it gives you anything different from COUNT(*)

Reply:
Interesting that despite of using filtering in `COUNT(x)` the `COUNT(*)` is even faster during tests.

> Leaving the choice up to the optimizer by using COUNT(*) lets it optimize based on conditions automatically.

Changing to `COUNT(*)`.

Reply:
[PostgreSQL doc reference](https://wiki.postgresql.org/wiki/Slow_Counting) regarding the `COUNT` and this topic.